app/views/documents/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= content_for :page_title, "New #{current_format.title.singularize}" %>
```

Comment:
I'm assuming the `singularize` method was used for changing a plural to singular. Do we have a problem with any that are named with plurals?